src/main/java/de/tum/in/www1/artemis/service/exam/ExamRegistrationService.java

```diff
@@ -263,6 +270,10 @@ public void unregisterStudentFromExam(Exam exam, boolean deleteParticipationsAnd
         examRepository.save(exam);
         examUserRepository.delete(registeredExamUser);
 
+        // Delete the signing and student image if they exist
+        Stream.of(registeredExamUser.getSigningImagePath(), registeredExamUser.getStudentImagePath()).filter(Objects::nonNull).map(URI::create)
+                .map(FilePathService::actualPathForPublicPath).forEach(path -> fileService.schedulePathForDeletion(path, 0));
+
```

Comment:
Whilst small, this is still duplicated, so a helper method here would be great. This also avoids the code comment by repacing it with a meaningful method name.